apps/aecore/test/aec_peers_tests.erl

```diff
@@ -44,10 +44,6 @@ all_test_() ->
                ?assertEqual(ok, aec_peers:remove("http://someone.somewhere:1337/v1")),
                ?assertEqual(1, length(aec_peers:all()))
        end},
-      {"Uri_from_ip_port",
-       fun() ->
-               ?assertEqual("http://123.123.123.123:1337/", aec_peers:uri_from_ip_port("123.123.123.123", 1337))
```

Comment:
I am trying to understand whether there is now an equivalent test for this piece of functionality that now moved to other app. (It appears to be called in sync suite maybe.)

Reply:
It was not a test other than checking that ++ works
I removed the function completely and hence no test that ++ works any more